shared/utils.sh

```diff
@@ -56,14 +57,23 @@ function get_metadata() {
     echo `curl -s http://169.254.169.254/latest/meta-data/$key`
 }
 
-function logthis() {
-    echo "[`date`] $1" >> $(get_config_value .logging.log_file)
+function loginfo() {
+    echo "INFO $1" >> $(get_config_value .logging.log_file)
```

Comment:
Will the log still have the timestamp since `date` is removed ?

Reply:
yea, adding timestamps is part of splunk docker log driver. having them in the message line was very noisy.